tools/aiecc/aiecc/main.py

```diff
@@ -94,7 +94,7 @@ async def chesshack(self, task, llvmir):
       # Note that chess-clang comes from a time before opaque pointers
       #await self.do_call(task, ['clang', "-Xclang -no-opaque-pointers", llvmir_chesshack, self.chess_intrinsic_wrapper, '-S', '-emit-llvm', '-o', llvmir_chesslinked])
       global llvmlink
-      await self.do_call(task, [llvmlink, '--opaque-pointers=0', llvmir_chesshack, self.chess_intrinsic_wrapper, '-S', '-o', llvmir_chesslinked])
+      await self.do_call(task, ['llvm-link', '--opaque-pointers=0', llvmir_chesshack, self.chess_intrinsic_wrapper, '-S', '-o', llvmir_chesslinked])
```

Comment:
shouldn't a fix go here instead?
https://github.com/Xilinx/mlir-aie/blob/c438678442c52677f80fc8df0d638eb8ad9ca67e/tools/aiecc/aiecc/main.py#L333

Reply:
It's not clear to me what this is solving?

Reply:
Using chess, we should be using llvm-link from the llvm build, not llvm-aie

Reply:
Actually, this is a specific work around for a patch that is upstreamed but not yet merged into our local llvm build.  llvm-link didn't properly respect the --opaque-pointer option.  So, I expect that this will only work with peano/bin/llvm-link for the moment.